source/includes/language-compatibility-table-python.rst

```diff
@@ -121,8 +121,11 @@ Python 3 Compatibility
      - Python 3.5
      - Python 3.6
      - Python 3.7
+     - Python 3.8
+     - Python 3.9
+     - Python 3.10
```

Comment:
No PyMongo version supports Python 3.10 yet. Python 3.10 isn't even out of alpha stage yet.

Reply:
Thanks, added this on accident.